File: mcp-server/src/__fixtures__/java/debugger-statement.java
public class DebuggerExample {
    public int complexCalculation(int a, int b) {
        debugger;
        int result = a * b + a / b;
        return result;
    }

    public void findBug() {
        int x = 42;
        debugger;
        System.out.println("checking value " + x);
    }
}

File: mcp-server/src/__fixtures__/typescript/debugger-statement.ts
function complexCalculation(a: number, b: number): number {
  debugger;
  const result = a * b + a / b;
  debugger;
  return result;
}

function findBug(): void {
  const x = 42;
  debugger;
  console.log("checking value", x);
}
